ui/main/src/app/business/view/navbar/navbarMenu.view.spec.ts

```diff
@@ -576,6 +575,17 @@ describe('NavbarMenuView', () => {
             expect(rightMenuElements[2].id).toEqual('useractionlogs');
         });
 
+        it('visible if configured in ui-menu config with user for admin feature useractionlogs', async () => {
+            await stubConfigLoading({
+                navigationBar: [],
+                topRightMenus: [{opfabCoreMenuId: 'useractionlogs', visible: true}]
+            });
+            await stubCurrentUserData([], [PermissionEnum.VIEW_USER_ACTION_LOGS]);
```

Comment:
Test is passing even is user has not permission VIEW_USER_ACTION_LOGS

It shoud not

Could you add a test that menu is not visible id user has no permission and have it passing